backend/open_webui/routers/scim.py

```
@@ -0,0 +1,926 @@
+import time
+from typing import Optional, List, Dict, Any
+from datetime import datetime, timezone
+
+from fastapi import APIRouter, Depends, HTTPException, Request, Query, Header, status
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel, Field, ConfigDict
+
+from open_webui.models.users import Users, UserModel
+from open_webui.models.groups import Groups, GroupModel
+from open_webui.utils.auth import (
+    get_admin_user,
```

Ruff (0.12.2)

`open_webui.utils.auth.get_admin_user` imported but unused

Remove unused import

(F401)